core/serialisation/data_serialiser.py

```diff
@@ -43,6 +43,19 @@
     risk_register_query,
 )
 
+# This project uses the Marshmallow library to define the serialization schema. Additional classes can be defined inside
```

Comment:
Please can you move this into the module level docstring? (that's basically the multi-line string at the top of the file)